source/features/reactions-avatars.tsx

```diff
@@ -56,8 +56,8 @@ async function showAvatarsOn(commentReactions: Element): Promise<void> {
 	const avatarLimit = arbitraryAvatarLimit - (commentReactions.children.length * approximateHeaderLength);
 
 	const participantByReaction = select
-		.all(':scope > button[aria-label$="emoji"]', commentReactions)
-		.map(button => getParticipants(button));
+		.all(':scope > button:is([aria-label$="emoji"], [title$="emoji"])', commentReactions) // `aria-label` is for PR/issue comments, `title` for releases
+		.map(button => getParticipants(button as HTMLButtonElement));
```

Comment:
Alternatively the selector could simply be
```css
button.social-reaction-summary-item
```
It's valid on PR/issues as well and doesn't necessitate the type coercion (which I guess is due to a bug/limitation of `typed-query-selector`?)

Reply:
Let's do this if it works

```suggestion
		.all(':scope > button.social-reaction-summary-item', commentReactions) // `aria-label` is for PR/issue comments, `title` for releases
		.map(button => getParticipants(button as HTMLButtonElement));
```

Reply:
I just hope this doesn't break things on GHE